src/webrtc/peerConnectionUtils.ts

```diff
@@ -0,0 +1,7 @@
+export const findSender = (
```

Comment:
As a person that reads code, I don't really see relation between file name and function inside

Reply:
You're right. I've change the name of this file to `RTCPeerConnectionUtils.ts` to indicate that this files contains utilities and helpers for `RTCPeerConnection` class. 

I think, in the future, I will introduce some kind of wrapper for `RTCPeerConnection` class if managed to extract more utilites.